RNWCPP/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-windows",
-  "version": "0.57.0-vnext.30",
+  "version": "0.58.0-vnext.0",
```

Comment:
"6" - Is there a reason we wouldn't just make this the same version as the fb one we pull in?

Reply:
This version will bump independently, based on changes made to RNW.